documents/customer.md

```diff
@@ -145,6 +145,244 @@ client.customer.fetch(customerId)
 
 -------------------------------------------------------------------------------------------------------
 
+### Add Bank Account of Customer
+
+```py
+customerId = "cust_N5mywh91sXB69O"
+
+client.customer.addBankAccount(customerId, {
+    "ifsc_code" : "UTIB0000194",
+    "account_number"         :"916010082985661",
+    "beneficiary_name"      : "Pratheek",
+    "beneficiary_address1"  : "address 1",
+    "beneficiary_address2"  : "address 2",
+    "beneficiary_address3"  : "address 3",
+    "beneficiary_address4"  : "address 4",
+    "beneficiary_email"     : "random@email.com",
+    "beneficiary_mobile"    : "8762489310",
+    "beneficiary_city"      :"Bangalore",
+    "beneficiary_state"     : "KA",
+    "beneficiary_country"   : "IN"
+})
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*  | string      | Unique identifier of the customer.  |
+| account_number | string      | The name of the beneficiary associated with the bank account.  |
+| beneficiary_name | string  | The virtual payment address.  |
+| beneficiary_address1 | string      | The id of the customer to be fetched  |
+| beneficiary_email  | string      | Email address of the beneficiary.  |
+| beneficiary_mobile | integer      | Mobile number of the beneficiary.  |
+| beneficiary_city  | string      | The name of the city of the beneficiary.  |
+| beneficiary_state | string      | The state of the beneficiary.  |
+| beneficiary_pin   | interger    | The pin code of the beneficiary's address.  |
+| ifsc_code          | string      | The IFSC code of the bank branch associated with the account.  |
```

Comment:
Please check the description of each of the parameters and make sure they are proper

Reply:
Go through the entire file and make sure the information is correct.

Reply:
made all doc correction in new added endpoint